cmds/exp/tcpdump/util.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/vishvananda/netlink"
+)
+
+var wellKnownPortsMap = map[string]string{
```

Comment:
can we at some point have a go generate step to create this? My favorite port, 17010, is not in here :-)

Reply:
I haven't looked much into `go generate` myself yet, what advantages would it bring? 
Do we have to write additional tooling for the CI, and so on for this? If so I would like to do this separately from this PR.
I am open to suggestions regarding well-known ports, although I don't know what port 17010 would be ;)